dev/com.ibm.ws.crypto.common/resources/com/ibm/ws/crypto/common/resources/CryptoCommonMessages.nlsprops

```diff
@@ -0,0 +1,28 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.crypto.common/resources/com/ibm/ws/crypto/common/resources/SamlSso20Messages.nlsprops
+#COMPONENTPREFIX CWWKS
+#COMPONENTNAMEFOR WebSphere Application Server Security Common Crypto Utilities
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+#Message prefix block: CWWKS5900 - CWWKS5910
+CRYPTO_UNSECURE=CWWKS5900W: Configuration element {0} specifies unsecure cryptographic algorithm {1}.  Consider using the secure algorithm {2} instead.
+CRYPTO_UNSECURE.explanation=The configured algorithm is no longer considered cryptographically secure.
+CRYPTO_UNSECURE.useraction=Use the suggested secure algorithm.
+
+CRYPTO_UNSECURE_REPLACED=CWWKS5901W: Configuration element {0} specifies unsecure cryptographic algorithm {1}.  The secure algorithm {2} is used instead."
```

Comment:
Remove end " and use the "the variable noun" format:

The {0} configuration element specifies the {1} unsecure cryptographic algorithm. The {2} secure algorithm is used instead.

Reply:
Thanks, Pam.  I updated it as you suggested.